client/cmd/bisonw-desktop/icon_windows.go

```diff
@@ -35,7 +35,7 @@ import (
 	"strings"
 	"unsafe"
 
-	"github.com/webview/webview"
```

Comment:
Unsure why my formatter (gofmt I think) is not putting "webview" in front of this import but does in other places. I guess either works with the "_go" suffix.